### DISCUSSION TITLE
LF4 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#143] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/LF4.html

Author: @tdegeorge 
Reviewer: @kathypinzon

**Comment by siwelwerd** [comment]
@kathypinzon I got Tonya's first draft merged in last night so it is ready for your review.

**Comment by kathypinzon** [comment]
Def 3.4.2 
I would break the definition up into a remark and a definition.
Remark: Notice that in [Activity 3.4.1] the two lines never touch.
Definition: I wouldn't put the "same slope" part as a term since that isn't really what we are defining.

**Comment by kathypinzon** [comment]
3.4.3 c I would remove the word "also" - it makes it seem like the line they have passes through that point too.

**Comment by kathypinzon** [comment]
3.4.4 (e) say "slope of each line" instead of "slopes"
3.4.4 (f) is there a better way to get at the intended result?  Maybe - how would you compare the slopes of the two lines?

**Comment by kathypinzon** [comment]
remark 3.4.5 you can make the size of the parentheses the same as the fraction by using \left( \right) instead of just ( )

**Comment by kathypinzon** [comment]
3.4.7 (c) same as 3.4.3 (c)

**Comment by kathypinzon** [comment]
3.4.8 (d) can you make the slopes something besides 2 and -1/2 (since you used those in the previous examples).  Both could be fractions even.

**Comment by tdegeorge** [comment]
@siwelwerd can you check to make sure this pull request went through okay? It gave me a weird message saying that there were no changes when I submitted it...

**Reply by siwelwerd** [reply]
If you click on 'Pull Requests' at the top you can always check to see.  And then you can click on [Files changed](https://github.com/TeamBasedInquiryLearning/precalculus/pull/212/files) to see what changes were made. So it looks like it went through okay
